src/info/langs/language.rs

```diff
@@ -245,7 +245,7 @@ define_languages! {
     { OCaml, Programming, "ocaml.ascii", define_colors!([Yellow]), Rgb(59, 225, 51) },
     { Org, Prose, "org.ascii", define_colors!([Green, Red, White]), Rgb(119, 170, 153) },
     { Perl, Programming, "perl.ascii", define_colors!([Cyan]), Rgb(2, 152, 195) },
-    { Php, Programming, "php.ascii", define_colors!([Magenta, Blue, Cyan, White]), Rgb(79, 93, 149) },
+    { Php, Programming, "php.ascii", define_colors!([Magenta, White] : [Rgb(119, 123, 179), Rgb(255, 255, 255)]), Rgb(79, 93, 149) },
```

Comment:
```suggestion
    { Php, Programming, "php.ascii", define_colors!([Blue, White] : [Rgb(119, 123, 179), Rgb(255, 255, 255)]), Rgb(79, 93, 149) },
```

More inline with the rgb color palette